test/spec/date.spec.js

```diff
@@ -1,64 +0,0 @@
-/**
- * @module date.spec
- * @see {@link https://github.com/medic/cht-core/blob/2c1db1618bdafc5ec14c3a27aa4a37249fcc1b4a/webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js}
- *
- * Note: These tests were copied from `medic/cht-core` as they appeared to be
- * the only relevant existing tests of this functionality. They've been adapted
- * to:
```

Comment:
Are these tests in the evaluator? If not, would be good to add them.

Reply:
Went one further and got us on track to stop relying on the prototype monkeypatch! https://github.com/enketo/openrosa-xpath-evaluator/pull/126